test/e2e/case23_compliance_api_recording_test.go

```diff
@@ -198,16 +199,19 @@ var _ = Describe("Compliance API recording", Ordered, Label("compliance-events-a
 				g.Expect(requests[0].Event.Message).To(Equal("Halt, who goes there?"))
 				g.Expect(requests[0].ParentPolicy.KeyID).To(BeEquivalentTo(1))
 				g.Expect(requests[0].Policy.KeyID).To(BeEquivalentTo(3))
+				g.Expect(requests[0].Event.Timestamp.Nanosecond()).ToNot(Equal(0))
```

Comment:
It's possible that this could actually be 0, though it's very unlikely since it'd be a 1 in 999999999 chance for each event. If the reviewer is spooked about this flake happening, I can remove the check.

Reply:
You mean there is a possibility of flaky test?

Reply:
Yes because it's possible the the actual nanosecond of when the event was generated was 0. I'll just change it to be a unit test instead.